### PR TITLE
feat: implement `clearGridSelections`

### PR DESCRIPTION
**Pull request by lja1018**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description

- implement `clearGridSelections`
- add tests for `clearGridSelections`

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/calendar/docs/ko/apis/calendar.stories.mdx [hunk 1]
```diff
@@ -593,6 +594,18 @@ calendar.openFormPopup({
 
 [⬆️ 목록으로 돌아가기](#인스턴스-메서드)
 
+### clearGridSelections
+
+- 타입: `clearGridSelections(): void`
+
+현재 캘린더에 표시되고 있는 날짜/시간 선택 엘리먼트를 제거한다.
```
Comment (adhrinae) [inline apps/calendar/docs/ko/apis/calendar.stories.mdx:601]:
```suggestion
현재 캘린더에 표시된 모든 날짜/시간 선택 엘리먼트를 제거한다.
```

File: apps/calendar/docs/ko/apis/calendar.stories.mdx [hunk 2]
```diff
@@ -89,7 +89,8 @@ calendar.on('beforeCreateEvent', (eventObj) => {
 | [setCalendarVisibility](#setcalendarvisibility) | 지정된 이벤트 그룹에 포함되는 모든 이벤트를 숨기거나 표시한다.                                                             |
 | [setCalendarColor](#setcalendarcolor)           | 지정된 이벤트 그룹에 포함된 모든 이벤트의 컬러 값을 변경한다.                                                              |
 | [scrollToNow](#scrolltonow)                     | 일간 뷰 혹은 주간 뷰에서 현재 시간이 포함된 범위를 표시하고 있는 경우, 즉시 현재 시간 영역으로 스크롤을 이동한다.          |
-| [openFormPopup](#openformpopup)                 | 이벤트를 생성하는데 사용하는 팝업을 표시한다. 전달한 객체의 정보에 따라 팝업의 값이 j워져 있다.                            |
+| [openFormPopup](#openformpopup)                 | 이벤트를 생성하는데 사용하는 팝업을 표시한다. 전달한 객체의 정보에 따라 팝업의 값이 채워져 있다.                           |
+| [clearGridSelections](#cleargridselections)     | 현재 캘린더에 표시되고 있는 날짜/시간 선택 엘리먼트를 제거한다.                                                            |
```
Comment (adhrinae) [inline apps/calendar/docs/ko/apis/calendar.stories.mdx:93]:
```suggestion
| [clearGridSelections](#cleargridselections)     | 현재 캘린더에 표시된 모든 날짜/시간 선택 엘리먼트를 제거한다.                                                            |
```

File: apps/calendar/src/factory/calendarControl.spec.tsx [hunk 3]
```diff
@@ -1371,3 +1372,156 @@ describe('getDateRangeStart/getDateRangeEnd', () => {
     });
   });
 });
+
+describe('clearGridSelections', () => {
+  const gridSelectionsTypes: GridSelectionType[] = ['dayGridMonth', 'dayGridWeek', 'timeGrid'];
+
+  describe.each(gridSelectionsTypes)('when %s grid selection', (gridSelectionType) => {
```
[the remark this answers is not in the document]
Comment (lja1018) [reply apps/calendar/src/factory/calendarControl.spec.tsx:1379]: 내부에서 `beforeEach`, `afterEach`를 사용하기 위해 사용했습니다!

[the remark this answers is not in the document]
Comment (adhrinae) [reply apps/calendar/src/factory/calendarControl.spec.tsx]: 아 이해했습니다.

**Review by adhrinae — APPROVED** [review]

**Pull request merged**